src/lib/InputButton/InputButton.svelte

```diff
@@ -129,12 +129,6 @@
     min-width: 0px;
   }
 
-  .button {
-    flex: 1;
-    min-width: 0px;
```

Comment:
is it safe to remove this min-width in new class?

Reply:
Adding it back as it might have unseen usecases